stm32cube/stm32g0xx/README

```diff
@@ -39,6 +39,5 @@ Patch List:
    *Changes from official delivery:
     -dos2unix applied
     -trailing white spaces removed
-    -added include guards to stm32g0xx_hal_hcd.h(STM32CubeG0 GH issue: 13)
```

Comment:
Thanks for the update. As it was not updated in the upstream hal, I think this line should still remain?

Reply:
right, there was a mistake for stm32g0 and stm32h7